src/main/java/com/google/codeu/data/Message.java

```diff
@@ -43,6 +44,20 @@ public Message(UUID id, String user, String text, long timestamp, String recipie
     this.recipient = recipient;
   }
 
+  public Message(UUID id, String user, String text, long timestamp, String recipient, String imageUrl) {
+    this.id = id;
+    this.user = user;
+    this.text = text;
+    this.timestamp = timestamp;
+    this.recipient = recipient;
+    this.imageUrl = imageUrl;
+  }
+
```

Comment:
But this blank line is fine - you added the lines above and below, and this line should definitely remain.